client/src/components/output.js

```diff
@@ -67,24 +69,11 @@ class Output extends React.Component {
     };
   }
 
-  friendInfo(name, number) {
```

Comment:
We saved the friendInfo state in Redux so no need to have the friendInfo method anymore